index.html

```diff
@@ -183,7 +215,7 @@ <h2 class="title-text">My Recent Works</h2>
            </ul>
            </div>
            <div class="actz">
-               <button class="clicks btnn" type="button">See Project</button>
+               <button class="clicks btnn dp" type="button">See Project</button>
            </div>
            
        </div>
```

Comment:
Please create a footer for the desktop view as it is defined in the design.
![footer-o](https://user-images.githubusercontent.com/35504149/172962062-307b9d65-500c-4f99-a941-064d5534e9ba.png)

Reply:
Footer created for the desktop view.